x-pack/plugins/cases/public/components/create/schema.tsx

```diff
@@ -77,6 +77,7 @@ export type FormProps = Omit<CasePostRequest, 'connector' | 'settings' | 'owner'
   fields: ConnectorTypeFields['fields'];
   syncAlerts: boolean;
   selectedOwner?: string | null;
+  [x: `customFields.${string}`]: string | boolean;
```

Comment:
Sorry, my mistake. I was confused with how the form lib works. The correct type is: `customFields: Record<string, unknown>`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/166969/commits/f3fb5a6a81f5235ee29c4f5da5a3efc4b791dd57